tests/e2e/test_perf_ingestion.py

```diff
@@ -1,94 +0,0 @@
-import copy
```

Comment:
What's the reason to remove this test?
Not obvious to me.

Reply:
It's testing that the job loader is processing a `performance_artifact` blob from a pulse job, so it calls `store_job_data` -> `JobLoader._load_job` -> `store_job_artifacts`.

But what's actually happening is we're only processing performance "artifacts" via parsing the log for PERFORMANCE_DATA lines. So the log parser calls `store_job_artifacts`.

There is one test in `test_performance_artifact_builder` to validate log parsing, but on second though perhaps I should keep this test file I deleted and change it so it calls `store_job_artifacts` (or otherwise improve the log parsing coverage).

Reply:
Actually, there are a lot of tests covering `store_performance_artifact`, which is called in `store_job_artifacts` for the `performance_data` artifact so maybe we're good here.